examples/ex02_reqrep.nim

```diff
@@ -10,7 +10,7 @@ proc requester() =
     elem = i/10
 
   echo "listen"
```

Comment:
```suggestion
  echo "connect"
```

Reply:
I agree it makes more sense to fix the inverted ``echo`` (or remove them entirely).